pkg/plugin/v3/scaffolds/webhook.go

```diff
@@ -19,6 +19,8 @@ package scaffolds
 import (
 	"fmt"
 
+	"sigs.k8s.io/kubebuilder/pkg/plugin/v3/scaffolds/internal/templates/config/kdefault"
```

Comment:
Nit: Sort the import. It needs to be at the end.

Reply:
Done